src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Services/SyntaxFacts/VisualBasicSyntaxFacts.vb

```diff
@@ -152,97 +152,93 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.LanguageServices
             Dim memberAccess = TryCast(node?.Parent, MemberAccessExpressionSyntax)
             Return memberAccess IsNot Nothing AndAlso memberAccess.Name Is node
         End Function
 
-        Public Function GetStandaloneExpression(node As SyntaxNode) As SyntaxNode Implements ISyntaxFacts.GetStandaloneExpression
+        Public Overrides Function GetStandaloneExpression(node As SyntaxNode) As SyntaxNode
             Return SyntaxFactory.GetStandaloneExpression(TryCast(node, ExpressionSyntax))
         End Function
 
-        Public Function GetRootConditionalAccessExpression(node As SyntaxNode) As SyntaxNode Implements ISyntaxFacts.GetRootConditionalAccessExpression
+        Public Overrides Function GetRootConditionalAccessExpression(node As SyntaxNode) As SyntaxNode
             Return TryCast(node, ExpressionSyntax).GetRootConditionalAccessExpression()
         End Function
 
-        Public Function IsNamedArgument(node As SyntaxNode) As Boolean Implements ISyntaxFacts.IsNamedArgument
+        Public Overrides Function IsNamedArgument(node As SyntaxNode) As Boolean
             Dim arg = TryCast(node, SimpleArgumentSyntax)
             Return arg?.NameColonEquals IsNot Nothing
         End Function
 
-        Public Function IsNameOfNamedArgument(node As SyntaxNode) As Boolean Implements ISyntaxFacts.IsNameOfNamedArgument
+        Public Overrides Function IsNameOfNamedArgument(node As SyntaxNode) As Boolean
             Return node.CheckParent(Of SimpleArgumentSyntax)(Function(p) p.IsNamed AndAlso p.NameColonEquals.Name Is node)
         End Function
 
-        Public Function GetNameOfParameter(node As SyntaxNode) As SyntaxToken? Implements ISyntaxFacts.GetNameOfParameter
+        Public Overrides Function GetNameOfParameter(node As SyntaxNode) As SyntaxToken?
             Return DirectCast(node, ParameterSyntax).Identifier?.Identifier
         End Function
 
-        Public Function GetDefaultOfParameter(node As SyntaxNode) As SyntaxNode Implements ISyntaxFacts.GetDefaultOfParameter
+        Public Overrides Function GetDefaultOfParameter(node As SyntaxNode) As SyntaxNode
             Return DirectCast(node, ParameterSyntax).Default
         End Function
 
-        Public Function GetParameterList(node As SyntaxNode) As SyntaxNode Implements ISyntaxFacts.GetParameterList
+        Public Overrides Function GetParameterList(node As SyntaxNode) As SyntaxNode
             Return node.GetParameterList()
         End Function
 
-        Public Function IsParameterList(node As SyntaxNode) As Boolean Implements ISyntaxFacts.IsParameterList
+        Public Overrides Function IsParameterList(node As SyntaxNode) As Boolean
             Return node.IsKind(SyntaxKind.ParameterList)
         End Function
 
-        Public Function ISyntaxFacts_HasIncompleteParentMember(node As SyntaxNode) As Boolean Implements ISyntaxFacts.HasIncompleteParentMember
-            Return HasIncompleteParentMember(node)
-        End Function
```

Comment:
vb does not allow a base type to implicitly implement interfaces declared on the derived type.  so tehse impl+forwarding methods were needed.  THey can no go since the base type does impl the interface.